### ISSUE TITLE
Customize diff tool

### ISSUE THREAD

**Issue by wayneljw** [Natizyskunk/vscode-sftp#103]
**Is this a similar or duplicate feature request?**
- [ ] I don't know. I will go check it.
- [x] No.

**Is your feature request related to a problem? Please describe.**
I would like to use “Beyond Compare” when using “diff with remote”. Most solutions I found on the Internet only allows customizing the diff tool for diff with the repository.

**Describe the solution you'd like**
Be able to setup to open a customized command to use own diff tool. (i.e. command line customization)

**Does this project help you?**
- [x] Yes. SFTP IS AWESOME!

**Comment by keremkayhan** [comment]
Another opinion for diff tool:  
I dont know it there is an option but seeing the local file on the left and the remote file on the right would be awsome. Thanks.

**Comment by victor-david** [comment]
@keremkayhan Yes, indeed. I always have to think twice when diffing with remote. For me, it makes sense to have the remote on the right since it's "farther" away from the local.
